js/src/workspaces/window.js

```diff
@@ -594,17 +594,30 @@
         var dfd = jQuery.Deferred(),
         module = $.viewer.annotationEndpoint.module,
         options = $.viewer.annotationEndpoint.options;
```

Comment:
Adding an `endpoint = _this.endpoint;` would solve the problem by eliminating the unintentional global variable. However, one must make sure to delete the `;` from line 595 and replace it with a comma.